x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_registry_uncommon.json

```diff
@@ -67,5 +67,5 @@
   "timeline_title": "Comprehensive Registry Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 7
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950